db/python/layers/analysis.py

```diff
@@ -596,10 +598,11 @@ async def get_analysis_runner_log(
         project_ids: list[int] = None,
         author: str = None,
         output_dir: str = None,
+        ar_guid: str = None,
```

Comment:
Do you think we should use those filters here to expand the usability here, though unrelated to your change so probably a bad idea haha